scheduler/views.py

```diff
@@ -1,2 +1,32 @@
+from datetime import timedelta
+
 from django.conf import settings
+from django.utils import timezone
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import authentication, permissions
+
+from .models import ReminderSchedule
+
+class ReminderCreate(APIView):
+    queryset = ReminderSchedule.objects.all()
+
+    def post(self, request, *args, **kwargs):
+        try:
+            recipient_id = request.data['contacts'][0]['wa_id']
+        except KeyError:
+            status = 400
+            message = {"contacts.0.wa_id": ["This field is required."]}
+            return Response(message, status=status)
```

Comment:
What about creating a serialiser for this, instead of handling the validation ourselves?

Reply:
I thought that would be unnecessarily complicated. We only care about the `wa_id` field and Turn sends that to us nested in a few other objects. I figured I'd still have to pull apart the objects in the serializer to do the validation anyway. This seemed easier and simpler to me.

Reply:
We've done simple serialiser in the past: https://github.com/praekeltfoundation/ndoh-hub/blob/develop/eventstore/serializers.py#L278-L279 , and it just ignores any additional fields we don't care about. But happy to do it this way as well.